--- a/tests/nanobind_example/CMakeLists.txt
+++ b/tests/nanobind_example/CMakeLists.txt
@@ -44,6 +44,7 @@         ${nanobind_example_upstream_SOURCE_DIR}/src/nanobind_example
         ${CMAKE_CURRENT_SOURCE_DIR}/pyext@pyext
         # Files
         ${CMAKE_CURRENT_SOURCE_DIR}/example.py
+        ${CMAKE_CURRENT_SOURCE_DIR}/scripts/example_added.py
         ${CMAKE_CURRENT_SOURCE_DIR}/scripts/__init__.py@scripts
 )
 
--- /dev/null
+++ b/tests/nanobind_example/scripts/example_added.py
@@ -0,0 +1,2 @@
+def new_greeting() -> str:
+    return "hello new"
